### PR TITLE
OCM-5829 | feat: Supressing help message when error is thrown

### PR DESCRIPTION
**Pull request by den-rgb**
[JIRA](https://issues.redhat.com/browse/OCM-5829) Suppressing help message when error is shown from linking multiple roles to the same org

### CODE REVIEW

**Comment by hunterkepley** [comment]
Ahhh I always wondered how to do this, nice

**Comment by den-rgb** [comment]
> Ahhh I always wondered how to do this, nice

Yeah It's a simple enough solution haha

File: cmd/link/ocmrole/cmd.go [hunk 1]
```diff
@@ -151,6 +151,7 @@ func run(cmd *cobra.Command, argv []string) (err error) {
 		}
 		r.Reporter.Errorf("Unable to link role arn '%s' with the organization id : '%s' : %v",
 			roleArn, orgAccount, err)
+		cmd.SilenceUsage = true
```
Comment (ciaranRoche) [inline cmd/link/ocmrole/cmd.go:154]: Is it worth setting this when defining the cmd :thinking: just wondering what is the use case for when we  do not want to silence the usage :thinking:

Comment (den-rgb) [reply cmd/link/ocmrole/cmd.go:154]: It'll only be silenced for that error the help prompt should still show up when putting in invalid params

**Review by robpblake — APPROVED** [review]
Nicely done @den-rgb . Thanks!

**Comment by AlexVulaj** [comment]
I believe this PR introduced a bug now where any commands that used to call `.RunE` on these commands will error. For example, the following line will now try to call a nil function. https://github.com/openshift/rosa/blob/master/cmd/create/accountroles/cmd.go#L220

**Comment by mikeshng** [comment]
I think this PR might have introduced a bug for cluster upgrade. Please see comment in https://issues.redhat.com/browse/OCM-6619 for more details. CC @den-rgb @AlexVulaj

**Pull request merged**